src/main/java/com/google/opensesame/servlets/MentorsServlet.java

```diff
@@ -14,9 +14,24 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     ArrayList<MentorObject> mentors = new ArrayList<MentorObject>();
-    mentors.add(new MentorObject("Obi", "Obi is awesome."));
-    mentors.add(new MentorObject("Sami", "Sami is fun."));
-    mentors.add(new MentorObject("Richi", "Richi is cool."));
+    ArrayList<String> projects = new ArrayList<String>();
+    projects.add("OpenSesame");
```

Comment:
I know this is mostly for demonstration until we get datastore up and running, but I just wanted to make sure that we were in agreement that project IDs will be the datastore-generated IDs and will not be the names of the projects.